nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/util/FlowDifferenceFilters.java

```diff
@@ -121,6 +121,20 @@ public static boolean isIgnorableVersionedFlowCoordinateChange(final FlowDiffere
         return false;
     }
 
+    /**
+     * Predicate that returns true if the difference can NOT be modified in running state, and false if it can.
+     */
+    public static Predicate<FlowDifference> FILTER_MODIFIABLE_IN_RUNNING_STATE = (fd) -> !isModifiableInRunningState(fd);
```

Comment:
@s9514171 The name of this variable is confusing. Your comment says, "true if the difference can NOT be modified".  Can you include NOT in the variable name?  Right now it sounds like the opposite.

`FILTER_NOT_MODIFIABLE_IN_RUNNING_STATE = (fd) -> isModifiableInRunningState(fd);`

Reply:
Yes, the variable named incorrectly and should be changed to FILTER_NOT_MODIFIABLE_IN_RUNNING_STATE
let me modify it.
Thanks 👍

Reply:
Added a commit for wrong variable naming, thanks for your review